drafts/25-xxx-specifications.txt

```diff
@@ -452,6 +452,8 @@ ew02. The token-list is not expanded.
 ew04. The processor produces a diagnostic message that includes
       the token-list.
 
+ew06. The processor shall reject a submitted program if the processor
+      encounters an `#error` directive during preprocessing.
```

Comment:
I believe I mostly wrote the specs as if the `#` is pronounced, so would read that last line as if it were "encounters a pound error (or hash error) directive during processing". So I used `a` instead of `an`.

```suggestion
      encounters a `#error` directive during preprocessing.
```

I notice also that we're not consistent through the doc about which we use. I filed #70 to suggest we be consistent in the final edit.